BedrockServer.cpp

```diff
@@ -1021,7 +1022,10 @@ void BedrockServer::runCommand(unique_ptr<BedrockCommand>&& _command, bool isBlo
                         } else {
                             SINFO("Conflict or state change committing " << command->request.methodLine << " on worker thread.");
                             if (_enableConflictPageLocks) {
-                                lastConflictPage = db.getLastConflictPage();
+                                lastConflictTable = db.getLastConflictTable();
+                                if (!SStartsWith(lastConflictTable, "journal")) {
```

Comment:
Maybe leave a comment explaining why we're doing this?

Reply:
Done